### PR TITLE
FEATURE: CL-9394 Execution of contribution metric

### PR DESCRIPTION
**Pull request by akloboucnik**

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/80/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/81/

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/82/

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/83/

**Comment by akloboucnik** [comment]
ok to test

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/84/

**Comment by akloboucnik** [comment]
ok to test

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/85/

File: src/execution.js [hunk 1]
```diff
@@ -178,16 +220,21 @@ const metricToDefinition = metric => ({ element: get(metric, 'objectUri')});
 
 export const mdToExecutionConfiguration = (mdObj) => {
     const { measures, categories, filters } = mdObj;
-    const factMetrics = map(filter(measures, m => m.type === 'fact'), generatedMetricDefinition);
-    const metrics = map(filter(measures, m => m.type === 'metric'), metric => {
+    const attributes = map(filter(categories, c => c.collection === 'attribute'), categoryToElement);
+    const contributionMetrics = map(
+        filter(measures, m => m.showInPercent),
+        partial(contributionMetricDefinition, head(filter(categories, c => c.collection === 'attribute')))
```
Comment (vaclavbohac) [inline src/execution.js]: `head(filter())` is basically `find()`, right?

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/86/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/87/

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/88/

**Comment by akloboucnik** [comment]
ok to test

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/89/

**Pull request merged**